roles/vagrant_kubernetes/tasks/main.yml

```diff
@@ -1,3 +1,21 @@
+- name: set iptable rules for nfs
```

Comment:
This should go to the `kube_source` role.

Vagrant-kubernetes is my own attempt at vagrantizing kubernetes, currently this role is unused by default.